NEWS

```diff
@@ -55,6 +55,44 @@ included in the vX.Y.Z section and be denoted as:
    (** also appeared: A.B.C)  -- indicating that this item was previously
                                  included in release version vA.B.C.
 
+3.1.4 -- February, 2019
+-----------------------
+
+- Fix compile error when configured with --enable-mpi-java and
+  --with-devel-headers.  Thanks to @g-raffy for reporting the issue
+  (** also appeared: v3.0.4).
+- Only use hugepages with appropriate permissions.  Thanks to Hunter
+  Easterday for the fix.
+- Fix possible floating point rounding and division issues in OMPIO
+  which led to crashes and/or data corruption with with very large
```

Comment:
Sorry to bump on this one more time: as we just found out in https://github.com/open-mpi/ompi/issues/6285#issuecomment-466645046, the data corruption fixes are not yet backported, only the crashes.
```suggestion
  which led to crashes with large
```

(~~very~~: magic ranges every 400 MB is not too large :) )

Reply:
We'll get the data corruption fix in.  I'm not worried about the relatively meaning of "very".  😄

Reply:
haha, yes that's ok with me :)
Thanks for pushing the DDT backports!